core/domain/platform_parameter_list.py

```diff
@@ -67,3 +68,8 @@ class ParamNames(enum.Enum):
     ParamNames.ANDROID_BETA_LANDING_PAGE,
     'This flag is for Android beta promo landing page.',
     platform_parameter_domain.FeatureStages.PROD)
+
+Registry.create_feature_flag(
+    ParamNames.BLOG_PROJECT,
+    'This flag is for blog project pages.',
+    platform_parameter_domain.FeatureStages.PROD)
```

Comment:
@seanlip deferring to you to confirm that you're okay with this feature going straight to prod without explicit testing being completed first.

Reply:
Yup this is fine (but the name/description need to be clearer, per comment above).

Reply:
Changed to BLOG_PAGES and added a description- 'This flag is for blog home page, blog author profile page and blog post page.